### DISCUSSION TITLE
Can we use className with View when using nativewind & solito

### DISCUSSION THREAD

**Discussion by rajat-arora084** [nandorojo/solito#486] [Q&A]
Currently, we can use styled HOC to render a View with tailwind support. But is it possible to render it directly with classname.
Ill also share the reason. As currently i have a react project with tailwind support so basically i want to convert it into solito app.
Till now, I am able to convert HTML into views & it is working fine. But the issue remains is that in React i have used classnames which are not working in Native applications.
So, basically i do not want to rewrite the styling part. Is there a way to do it?

**Comment by nandorojo** [comment]
we should pass className down directly, i’ll consider adding this soon

**Reply by rajat-arora084** [reply]
classNames are not supported right now. I believe for them to be applicable we might need to have tailwind configuration applied on our codebase which will be applying the styling using babel. But as per the documentation, the babel is not yet supported by solito. So, is there something that i am missing. Or if that is something that is going to be available in future, then can you please share the roadmap as to when we can expect this to happen as i am waiting for this for my own project & i am looking forward to using it.

**Reply by nandorojo** [reply]
yeah i think react native web doesn’t support it actually. we can maybe work around it
